app/controllers/api/v1/recipes/favorites_controller.rb

```diff
@@ -1,4 +1,14 @@
 class Api::V1::Recipes::FavoritesController < ApplicationController
+  def index
+    user_id = params[:user_id]
+    favorite_recipes = FavoritesFacade.get_favorite_recipes(user_id)
+    if favorite_recipes[:errors]
```

Comment:
Looks great! I think part 2 of the project we can spend some time as the BE team going through all of our repos and abstracting some of our error handling out of controllers where possible if we want to. I know there's a few across our different repos that we can probably handle more cleanly, but ultimately not super important to do this early on!